package.json

```diff
@@ -165,7 +165,7 @@
     "react-native-image-picker": "^7.1.2",
     "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#cb392140db4953a283590d7cf93b4d0461baa2a9",
     "react-native-key-command": "^1.0.8",
-    "react-native-keyboard-controller": "1.15.2",
+    "react-native-keyboard-controller": "1.16.6",
```

Comment:
We can not merge this, because 1.16.6 gives a different selection coordinates

We have a separate PR with bump here: https://github.com/Expensify/App/pull/57580

Reply:
I have bumped on the PR to see what is next there